dotnet/samples/ApplicationInsightsExample/RepoUtils/RepoFiles.cs

```diff
@@ -6,13 +6,13 @@
 internal static class RepoFiles
 {
     /// <summary>
-    /// Scan the local folders from the repo, looking for "samples/skills" folder.
+    /// Scan the local folders from the repo, looking for "/prompts/samples" folder.
     /// </summary>
-    /// <returns>The full path to samples/skills</returns>
+    /// <returns>The full path to prompts/samples</returns>
```

Comment:
Thoughts on `plugins/samples`?

Reply:
I wanted to call this "prompts" so we could have a language agnostic folder in our repo. Calling it "plugins" could imply that it could be home to plugins with C# or Python code. In the future, this folder can also hold common prompts across languages (e.g., the prompts for planners).